### PR TITLE
[RestResourceIncidnet] Fix a bug that can't post comments

### PR DESCRIPTION
**Pull request by ashie**
Sorry, I forgot to add tests for it...

### CODE REVIEW

File: server/test/testFaceRestIncident.cc [hunk 1]
```diff
@@ -226,6 +226,57 @@ void test_postIncidentForUnknownEvent(void)
...
+			       expectedIncident.unifiedEventId));
+	RequestArg arg(path);
+	arg.userId = findUserWith(OPPRVLG_GET_ALL_SERVER);
+	arg.request = "PUT";
+	incidentInfo2StringMap(expectedIncident, arg.parameters);
+	arg.parameters["comment"] = "Assigned to @Mnagakawa";
+
+	// check the response
+	getServerResponse(arg);
+	string expectedResponse(
```
Comment (cosmo0920) [inline server/test/testFaceRestIncident.cc]: Initializing string style to be mixed in same test case is not good.
Use 

```cpp
string hoge = "piyo";
```

style instead.

Comment (cosmo0920) [inline server/test/testFaceRestIncident.cc]: Hmm..., both of constructor and = operator styles are valid.
Let's leave it here as-is.

**Comment by cosmo0920** [comment]
LGTM
Can I merge this?

**Comment by cosmo0920** [comment]
I've merged it!

**Comment by ashie** [comment]
Thanks!

**Pull request merged**